mypy.ini

```diff
@@ -0,0 +1,29 @@
+[mypy-nacl]
```

Comment:
Please merge `mypy.ini` and `pytest.ini` into `setup.cfg`.
The single config file is better than two :)

Reply:
Better indeed :)